packages/@react-spectrum/color/src/ColorArea.tsx

```diff
@@ -13,6 +13,16 @@
 import {classNames, dimensionValue, useFocusableRef, useStyleProps} from '@react-spectrum/utils';
 import {ColorThumb} from './ColorThumb';
 import {FocusableRef} from '@react-types/shared';
+import {
+  generateHSB_B,
+  generateHSB_H,
+  generateHSB_S,
+  generateHSL_H,
+  generateHSL_S,
+  generateRGB_B,
+  generateRGB_G,
+  generateRGB_R
+} from './Gradients';
```

Comment:
nit: filename should be lower case.

Also, perhaps for later, I suspect these should be in react-aria since they aren't spectrum specific and there's quite a bit of logic to replicate in a custom implementation.